technical-writing-one/readme.md

```diff
@@ -0,0 +1,37 @@
+# Technical Writing One
+
+Participants: [Pre-class prep, 1 hour](https://developers.google.com/tech-writing/one)
+
+If you haven't done the prep, you can't do this workshop, so go and find something else to get on with.
+
+Facilitators: [Pre-class prep, 1 hour](https://developers.google.com/tech-writing/for-instructors/one/instructors-guide)
```

Comment:
Maybe put the Participant and Facilitators instructions into distinct boxes. I find it a little distracting that, at first glance, the two hyperlinks look like they go to the same place labeled "Pre-class prep, 1 hour". I didn't find a pre-existing sibling workshop with a style to match against, so you can use your own judgement.